src/scos_tekrsa/__init__.py

```diff
@@ -1 +1 @@
-__version__ = "5.0.2"
```

Comment:
Should this be a major version change due to breaking changes in TekRSASigan constructor and acquire_time_domain_samples method signatures?

Reply:
Yes, bumped to 6.0.0 in cdd9f1871ee1aa0643dc98ad0c6a4cf19f5098a9